src/site-switcher/siteSwitcherComponent.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Controller for site switcher component
+ * @memberof miqStaticAssets
+ * @ngdoc controller
+ * @name SiteSwitcherController
+ */
+export class SiteSwitcherController {
```

Comment:
Please add `sites` attribute to this class, and might be good to add some interface to describe what properties is needed in site, no need to create custom file, just add it above controller's class.
```typescript
export interface Site {
  url: string;
  site: string;
  tooltip?: string;
  iconClass? any;
}
```